hw/ip/otbn/doc/dv/index.md

```diff
@@ -141,6 +141,18 @@ That is, we expect to see execution with each bit of each immediate field being
 We also expect to see each field with values `'0` and `'1` (all zeros and all ones).
 If the field is treated as a signed number, we also expect to see it with the extremal values for its range (just the MSB set, for the most negative value; all but the MSB set, for the most positive value).
 
+> The code to track this is split by encoding-schema in `otbn_env_cov`.
+> Each instruction listed below will specify its encoding schema.
+> Each encoding schema then has its own covergroup.
+> Rather than tracking toggle coverage as described above, we just track extremal values in a coverpoint.
+> This also implies toggle coverage for both signed and unsigned fields.
+> For unsigned fields of width `N`, the extremal values are `0` and `(1 << N) - 1`, represented by the bits `'0` and `'1` respectively.
+> For signed fields of width `N+1`, the extremal values are `-(1 << N)` and `(1 << N) - 1`.
+> These are represented by `{1'b1, {N{1'b0}}}` and `{1'b0, {N{1'b1}}}`: again, these toggle all the bits.
+> For example, `beq` uses the `B` schema, which then maps to the `enc_b_cg` covergroup.
+> This encoding schema's `OFF` field is tracked with the `off_cp` coverpoint.
+> Finally, the relevant cross is called `off_cross`.
+
 For any instruction that reads from or writes to a GPR, we expect to see that operand equal to `x0`, `x1` and an arbitrary register in the range `x2 .. x31`.
```

Comment:
Why X0? it restricts observability and coverage. It seems more of a special case.

Reply:
There are three types of GPR in OTBN: `x0` (the zeros register), `x1` (the call stack) and `x2 .. x31` (everything else). This sentence says that we expect to see each type used.